.github/workflows/claude.yml

```diff
@@ -20,21 +27,46 @@ jobs:
       id-token: write
       actions: read # Required for Claude to read CI results on PRs
     steps:
-      - name: Checkout repository
+      - name: 📂 Checkout code
         uses: actions/checkout@v5
         with:
           # Fix: Checkout PR branch for issue_comment events instead of default branch
           ref:
             ${{ github.event.issue.pull_request && format('refs/pull/{0}/head',
             github.event.issue.number) || github.ref }}
           fetch-depth: 0 # Get more history for better context
+          fetch-tags: true
 
-      - name: Configure git for Claude
+      - name: 🔧 Configure git for Claude
         run: |
           git config --global user.name "Claude Code"
           git config --global user.email "claude@anthropic.com"
 
-      - name: Run Claude Code
+      # Install essential tools for the PRQL project
+      - name: 🔧 Setup Task
+        uses: arduino/setup-task@v2
```

Comment:
FYI, they released a new GHA to setup task recently.
https://github.com/go-task/setup-task